#Default error message
exception.default=Error while processing the request.

#Organization role related messages
role.not.found= Role not found {0}.
role.already.found=Role {0} already exists.
exception.add.role=Exception occurred while saving organization role.
exception.update.role=Exception occurred while updating organization role.
exception.delete.role=Exception occurred while deleting organization role.
exception.fetching.role=Exception occurred while fetching role {0}.
exception.fetching.role.name=Exception occurred while fetching role(s) by name {0}.
exception.fetching.role.type=Exception occurred while fetching role(s) by type {0}.
exception.fetching.roles=Exception occurred while fetching role(s).
role.type.not.supported=Role type is not supported.
exception.role.creation.type=Role can't be create with role type.
exception.role.creation.not.supported.type=Role creation is not supported with type {0}.
exception.role.creation=Exception occurred while creating role.
exception.importing.roles=Exception occurred while importing role(s).
exception.fetching.users.role=Exception occurred while fetching user(s) of role {0}.
role.delete.success=Role {0} deleted successfully.
roles.import.success=Role(s) imported successfully.
role.creation.success=Role created successfully.
role.updated.success=Role {0} updated successfully.
role.updated.partial.error=Role assignment failed to some users during update, please check log file.
role.creation.partial.success=Role assignment failed to some users during creation, please check log file.
exception.fetching.internal.roles=Exception occurred while fetching internal role(s).
invalid.role.type=Role type {0} is invalid.
exception.fetching.role.count=Exception occurred while fetching total count of organization role(s).
external.role.cannot.delete=Role cannot be deleted, users are assigned to it.
sync.roles.sucess=Role(s) synced successfully.
roles.already.sync=Role(s) already synced.
exception.sync.roles=Exception occurred while syncing role(s).
exception.sync.role=Exception occurred while syncing role {0}.
exception.sync.internal.role=Exception occurred while syncing internal roles.
exception.update.sync.date=Exception occurred while updating sync date.

#Organization user related messages
user.not.found=User not found {0}.
manager.not.found=Manager not found {0}.
exception.add.user=Exception occurred while saving organization user.
exception.update.user=Exception occurred while updating organization user.
exception.delete.user=Exception occurred while deleting organization user.
exception.fetching.user.hierarchy=Exception occurred while fetching user hierarchy.
exception.fetching.user=Exception occurred while fetching user {0}.
exception.fetching.users=Exception occurred while fetching user(s).
exception.fetching.manager.user=Exception occurred while fetching manager of user {0}.
exception.fetching.hierarchy.depth=Exception occurred while fetching hierarchy.
exception.assigning.manager=Exception occurred while assigning manager to user.
exception.importing.users=Exception occurred while importing user(s).
exception.importing.user=Exception occurred while importing user {0}.
exception.assign.roles.user=Exception occurred while assigning roles to user {0}.
exception.fetching.assigned.roles.user=Exception occurred while fetching assigned role(s) of user {0}.
user.delete.success=User {0} deleted successfully.
users.import.success=User(s) imported successfully.
manager.assigned.success=Manager assigned to users successfully.
manager.assigned.partial.error=Manager assignment failed to some users, please check log file.
exception.fetching.peers.subordinates=Error occurred while fetching peer(s) and subordinate(s).
exception.fetching.logged.in.user.roles=Error occurred while fetching logged in user role(s).
exception.sync.users=Exception occurred while syncing user(s).
exception.sync.user=Exception occurred while syncing user {0}.
sync.users.success=User(s) synced successfully.
users.already.sync=User(s) were last synced at {0}. Please try to sync after {1}.
user.subordinates.deleted.success=User and subordinate(s) deleted successfully.
exception.fetching.last.sync.date=Exception occurred while fetching last sync date.
exception.fetching.rbac.users=Exception occurred while fetching security provider user(s).
manager.cannot.assign.success=Manager cannot assign, its a predefined hierarchy structure.
exception.updating.root=Exception occurred while updating ROOT.
exception.fetching.last.update.date=Exception occurred while fetching last updated date.
exception.assigning.subordinates=Exception occurred while assigning subordinate(s).
subordinates.assigned.success=Subordinate(s) assigned successfully. 
subordinates.assigned.partial.error=Subordinate(s) assignment failed to some users, please check log file.
invalid.hierarchy=Not able to perform requested operation as it may lead to inconsistent user hierarchy.
exception.predefine.hierarchies=Exception occurred while cleaning predefined hierarchies.
exception.sync=Exception occurred while fetching last sync date.
exception.sync.update=Exception occurred while updating sync date.
exception.import.subordinates=Exception occurred while updating subordinates.
exception.import.manager=Exception occurred while fetching manager.
exception.search.user=Exception occurred while searching user.

#organization realm related message
exception.fetching.realms=Exception occurred while fetching realm(s).
exception.add.realm=Exception occurred while creating realm {0}.
realm.created.success=Realm {0} created successfully.
realm.already.found=Realm {0} already exists.
realm.already.found.sp=Realm {0} already exists in security provider.
